src/indra_cogex/apps/curator/explorer_blueprint.py

```diff
@@ -505,18 +568,19 @@ class PaperForm(FlaskForm):
         validators=[DataRequired()],
         description="""\
             This field accepts identifiers from PubMed, PubMed Central, and DOI
-            as either a CURIEs that looks like <code>pubmed:1234</code>,
-            <code>pmc:PMC3084216</code>, or <code>doi:10.1038/nbt1156</code>
+            as either a CURIEs that looks like <code>pubmed:26186194</code>,
+            <code>PMC4617211</code>, or doi:<code>10.1016/J.CELL.2015.06.043</code>
             <strong>or</strong> as a local unique identifier that looks like
-            <code>1234</code>, <code>PMC3084216</code>, or
-            <code>10.1038/nbt1156</code>.
+            <code>26186194</code>, <code>PMC4617211</code>, or
+            <code>10.1016/J.CELL.2015.06.043</code>.
```

Comment:
I can only get the pubmed ID to work when I try this locally. For the DOI I get 404, probably because it contains `/`, but this should be an easy fix (see comment for `/entity/<prefix>:<identifier>` above). For the PMC ID, I get a timeout as it just loads for a very long time and never resolves for me (I waited for ~20 min before I killed the app).

Reply:
If I resovle the DOI problem as suggested in https://github.com/gyorilab/indra_cogex/pull/188/files#r1876655899, I instead get a timeout, here as well. I'm not sure if this happens for you too. How long does the PMCID search and the DOI search take for you?

Reply:
OK, the timeouts seem to be resolved after running this a second time, not sure why it happened the first time. Maybe the graph db was very busy.